{
  "@type": "build",
  "@href": "/v3/build/315440349",
  "@representation": "standard",
  "@permissions": {
    "read": true,
    "cancel": false,
    "restart": false
  },
  "id": 315440349,
  "number": "1143",
  "state": "failed",
  "duration": 254,
  "event_type": "push",
  "previous_state": "failed",
  "pull_request_title": null,
  "pull_request_number": null,
  "started_at": "2017-12-12T16:22:28.000Z",
  "finished_at": "2017-12-12T16:26:42.000Z",
  "private": false,
  "repository": {
    "@type": "repository",
    "@href": "/v3/repo/36486",
    "@representation": "minimal",
    "id": 36486,
    "name": "async-http-client",
    "slug": "AsyncHttpClient/async-http-client"
  },
  "branch": {
    "@type": "branch",
    "@href": "/v3/repo/36486/branch/2.0",
    "@representation": "minimal",
    "name": "2.0"
  },
  "tag": null,
  "commit": {
    "@type": "commit",
    "@representation": "minimal",
    "id": 93652219,
    "sha": "8eb03926b5fab580225716e44a1fd921197875fb",
    "ref": "refs/heads/2.0",
    "message": "Perform WriteListener#abortOnThrowable whatever the channel state, close #1488\n\nMotivation\n\nFor some reason lost in time, abortOnThrowable is only performed if the\nchannel is not a new one.\nI have no idea why and it swallow exceptions, so we get timeouts\ninstead of actual exception.\n\nModification:\n\nDrop this test and always perform abortOnThrowable, whatever the\nchannel state.\n\nResult:\n\nProper exception reported",
    "compare_url": "https://github.com/AsyncHttpClient/async-http-client/compare/482ad4b53c9e...8eb03926b5fa",
    "committed_at": "2017-12-12T16:21:41.000Z"
  },
  "jobs": [
    {
      "@type": "job",
      "@href": "/v3/job/315440357",
      "@representation": "minimal",
      "id": 315440357
    }
  ],
  "stages": [

  ],
  "created_by": {
    "@type": "user",
    "@href": "/v3/user/48956",
    "@representation": "minimal",
    "id": 48956,
    "login": "slandelle"
  },
  "updated_at": "2019-04-10T08:47:27.682Z"
}